backend/src/main/java/com/festago/fcm/application/MockFcmClient.java

```diff
@@ -0,0 +1,21 @@
+package com.festago.fcm.application;
+
+import com.festago.fcm.domain.FCMChannel;
+import com.festago.fcm.dto.FcmPayload;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile({"!prod", "!dev"})
```

Comment:
요러면 모든 환경에서 등록되니 
```suggestion
@Profile({"!prod & !dev"})
```
요걸 의도하신 거겠죠?

Reply:
매의 눈 푸우 22 감사합니다

Reply:
매눈푸 ㄷㄷㄷ